todocore/src/main/java/com/knighttodo/todocore/exception/FindDayTodoOrderIdException.java

```diff
@@ -0,0 +1,9 @@
+package com.knighttodo.todocore.exception;
+
+public class FindDayTodoOrderIdException extends RuntimeException{
```

Comment:
Add ResponseCode annotation (or something like this) with an appropriate Http code. But I'm not sure why do you have this exception at all